frontend/src/pages/projects/notebook/NotebookStatusText.scss

```diff
@@ -3,11 +3,11 @@
 .odh-notebook-status-popover {
   &__starting-text {
     cursor: pointer;
-    text-decoration: underline var(--pf-v6-global--BorderWidth--sm) dashed
-      var(--pf-v6-global--BorderColor--200);
+    text-decoration: underline var(--pf-t--global--text-decoration--width--default) dashed
```

Comment:
Replace underline and dashed to text-decoration tokens or help-text tokens (double check context)